packages/cli/src/services/orchestration.handler.base.service.ts

```diff
@@ -29,7 +17,6 @@ export abstract class OrchestrationHandlerService {
 	}
 
 	async shutdown() {
-		await this.redisSubscriber?.destroy();
```

Comment:
Who takes care of destroying the subscriber now?

Reply:
Here:

https://github.com/n8n-io/n8n/blob/9543d88dcea6ac1c5c0117ee32d21281e978c703/packages/cli/src/services/orchestration.service.ts#L84-L94